### PR TITLE
Add MUSAN data augmentation to LibriSpeech MMI recipe

### PR DESCRIPTION
**Pull request by pzelasko**
Corresponding Lhotse PR: https://github.com/lhotse-speech/lhotse/pull/180

### CODE REVIEW

**Comment by pzelasko** [comment]
Will update about results tomorrow.

**Comment by pzelasko** [comment]
WER: 
```
2021-01-12 08:59:55,105 INFO [mmi_bigram_decode.py:297] %WER 10.07% [5294 / 52576, 745 ins, 506 del, 4043 sub ]
```

Loss:
```
2021-01-12 03:21:54,212 INFO [mmi_bigram_train.py:244] Validation average objf: 0.150381 over 645831.0 frames (100.0% kept)
2021-01-12 03:22:00,970 INFO [mmi_bigram_train.py:219] batch 2610, epoch 9/10 global average objf: 0.145567 over 25475184.0 frames (100.0% kept), current batch average objf: 0.189775 over 9796 frames (100.0% kept) avg time waiting for batch 0.004s
2021-01-12 03:22:07,895 INFO [mmi_bigram_train.py:219] batch 2620, epoch 9/10 global average objf: 0.145614 over 25573083.0 frames (100.0% kept), current batch average objf: 0.152133 over 9827 frames (100.0% kept) avg time waiting for batch 0.004s
2021-01-12 03:22:14,702 INFO [mmi_bigram_train.py:219] batch 2630, epoch 9/10 global average objf: 0.145648 over 25670957.0 frames (100.0% kept), current batch average objf: 0.183924 over 9901 frames (100.0% kept) avg time waiting for batch 0.004s
2021-01-12 03:22:21,415 INFO [mmi_bigram_train.py:219] batch 2640, epoch 9/10 global average objf: 0.145620 over 25767906.0 frames (100.0% kept), current batch average objf: 0.136576 over 9519 frames (100.0% kept) avg time waiting for batch 0.004s
2021-01-12 03:22:28,086 INFO [mmi_bigram_train.py:219] batch 2650, epoch 9/10 global average objf: 0.145598 over 25865185.0 frames (100.0% kept), current batch average objf: 0.115883 over 9863 frames (100.0% kept) avg time waiting for batch 0.004s
2021-01-12 03:22:29,497 INFO [common.py:93] Save checkpoint to exp-lstm-adam-mmi-bigram-musan/best_model.pt: epoch=9, learning_rate=0.0005120000000000001, objf=0.14559383016339622
```

The previous WER was in the range 11.09~11.19%, the improvement looks solid.

**Comment by csukuangfj** [comment]
👍

**Comment by danpovey** [comment]
Fantastic!
